spec/RECORDS.md

```diff
@@ -0,0 +1,22 @@
+# WFDB Records
+
+A WFDB record contains a continuous recording from a single subject.
+
+## Naming
+
+Records are identified by record names of up to 50 characters. A record name is case-sensitive, and must only contain letters, digits, and underscores.
+
+## Content
+
+There is no single 'record file'. A record is comprised of potentially several files:
+
+- One header file. The `record name` field is specified in the header file, and the file itself must be named: `<record_name>.hea`. See the [headers page](./HEADERS.md) for details.
+- Zero or more signal files. The names of the signal files are specified in the header file. The first part of each file name is not required to, but does conventionally match the record name. ie. `<record_name>.dat` or `<record_name>_1.dat`. See the [signals page](./SIGNALS.md) for details.
+
+A record may also have zero or more associated annotation files. However, the header file does not have any structured fields which specify associated annotation files, therefore the annotations are not considered _part of_ the record. Annotation files are identified to be associated with a record primarily by having the first parts of their file names fully matching that of the record. ie. `<record_name>.qrs`. See the [annotations page](./ANNOTATIONS) for details.
```

Comment:
My main reference was this page: https://physionet.org/physiotools/wpg/wpg_2.htm#Concepts-1

But I disagree with this sentence: "A record is comprised of several files, which contain signals, annotations, and specifications of signal attributes;" regarding the annotations. I think we need to more clearly define the concept of what a record is exactly.

Perhaps it should just be the description of the patient to some limited extent, and the signals? If there's a 12 hour recording of a patient's ECG written to a header and dat file, I believe that that's a record. If I come along and run a qrs detector on the signals 10 years later, I don't believe that they should qualify as being part of the record.